prometheus/collect_acl_info.go

```diff
@@ -0,0 +1,69 @@
+package prometheus
+
+import (
+	"context"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/twmb/franz-go/pkg/kmsg"
+	"go.uber.org/zap"
+)
+
+func (e *Exporter) collectACLInfo(ctx context.Context, ch chan<- prometheus.Metric) bool {
```

Comment:
Should only be done if configured (config may default to true, since the extra cardinality is relatively low)

Reply:
Thanks @weeco for the quick reply , I have updated the code to reflect this